theoplayer/web/hosted-player.md

```diff
@@ -56,14 +56,14 @@ The following parameters can be provided used to configure the hosted player. _N
 
 The following parameters are only valid with Dolby OptiView real-time streaming and will be ignored with other `sourceTypes`.
 
-- `minDelay`: _(default: `50`)._ a small minimum playout delay set to improve performance in degraded network conditions. Specified in milliseconds. See [Documentation](https://docs.optiview.dolby.com/millicast/playback/players-sdks/web/player/#playout-delay).
+- `minDelay`: _(default: `50`)._ a small minimum playout delay set to improve performance in degraded network conditions. Specified in milliseconds. See [Documentation](https://optiview.dolby.com/docs/millicast/playback/players-sdks/web/player/#playout-delay).
```

Comment:
Nit-pick: if you strip off the base URL, it'll also work correctly in pull request previews. 😉

```suggestion
- `minDelay`: _(default: `50`)._ a small minimum playout delay set to improve performance in degraded network conditions. Specified in milliseconds. See [Documentation](/millicast/playback/players-sdks/web/player/#playout-delay).
```